keen/explorer
# Fix issue #168

### What does this PR do?

Resolves issue https://github.com/keen/explorer/issues/168 ensuring the percentile parameter is saved and executed as a number (not a string). Goes with bug fix: https://github.com/keen/explorer/pull/173.

### How can I test this?

Run a percentile query and check out the Request Payload in your browser (Network tab in Chrome). The percentile value should be a number.
